### PR TITLE
[Xedra Evolved] Misc additions

### PR DESCRIPTION
**Pull request by GuardianDll**
#### Summary
None
#### Purpose of change
Add a use case for #68772
#### Describe the solution
Add 8 new gunmods, that utilizes overheat mechanic and changes it depending on gunmods
While here, fixed some bugs:
Inventor grenades has way too big crafting time, decreased it
Fixed the heavy modified battery description
Fixed blocky pistol being not rigid
Fixed inventor's grenade launcher jammed every time (in theory)
#### Additional context
Didn't know github can create pull requests right from github dev. Apparently they are created empty

### CODE REVIEW

File: data/mods/Xedra_Evolved/items/inventor/gunmods.json [hunk 1]
```diff
@@ -165,5 +165,149 @@
...
+    "mod_targets": [ "pistol", "rifle", "smg" ],
+    "cooling_value_multiplier": 0.4,
+    "ammo_to_fire_multiplier": 1.25,
+    "min_skills": [ [ "deduction", 4 ], [ "weapon", 4 ] ]
+  },
+  {
+    "id": "inventor_lasmod_recuperator",
+    "type": "GUNMOD",
+    "name": { "str_sp": "heat recuperator" },
+    "description": "Put some amount of the heat exchange from the gun back into platform loop.  Decrease amount of energy used, and increase the heat output of the weapon, but also it's heat generation within the loop.",
```
Comment (TheShadowFerret) [inline data/mods/Xedra_Evolved/items/inventor/gunmods.json:208]:
```suggestion
    "description": "Put some amount of the heat exchange from the gun back into platform loop.  Decrease amount of energy used, and increase the heat output of the weapon, but also its heat generation within the loop.",
```

File: data/mods/Xedra_Evolved/items/inventor/gunmods.json [hunk 2]
```diff
@@ -165,5 +165,149 @@
...
+    "location": "radiator",
+    "mod_targets": [ "pistol", "rifle", "smg" ],
+    "overheat_threshold_modifier": 150,
+    "min_skills": [ [ "deduction", 4 ], [ "weapon", 4 ] ]
+  },
+  {
+    "id": "inventor_lasmod_spooky_heat_sink",
+    "type": "GUNMOD",
+    "name": { "str_sp": "skeletonized heat sink" },
+    "description": "Replace the big part of radiator with additional cooling unit.  The gun overheat faster, but recovers from it at a faster rate",
```
Comment (TheShadowFerret) [inline data/mods/Xedra_Evolved/items/inventor/gunmods.json:281]:
```suggestion
    "description": "Replace a large part of the weapon's radiator with an additional cooling unit.  The gun overheats faster, but recovers from it at a faster rate.",
```

File: data/mods/Xedra_Evolved/items/inventor/gunmods.json [hunk 3]
```diff
@@ -165,5 +165,149 @@
...
+    "material": [ "steel" ],
+    "symbol": ":",
+    "color": "light_gray",
+    "location": "radiator",
+    "mod_targets": [ "pistol", "rifle", "smg" ],
+    "overheat_threshold_modifier": 150,
+    "min_skills": [ [ "deduction", 4 ], [ "weapon", 4 ] ]
+  },
+  {
+    "id": "inventor_lasmod_spooky_heat_sink",
```
Comment (TheShadowFerret) [inline data/mods/Xedra_Evolved/items/inventor/gunmods.json:278]: I kek'd

File: data/mods/Xedra_Evolved/items/inventor/gunmods.json [hunk 4]
```diff
@@ -165,5 +165,149 @@
...
+    "mod_targets": [ "pistol", "rifle", "smg" ],
+    "heat_per_shot_multiplier": 0.8,
+    "overheat_threshold_multiplier": 0.8,
+    "min_skills": [ [ "deduction", 4 ], [ "weapon", 4 ] ]
+  },
+  {
+    "id": "inventor_lasmod_more_capacitors",
+    "type": "GUNMOD",
+    "name": { "str_sp": "additional capacitors" },
+    "description": "Replace the big part of the radiator with additional capacitors.  The gun barely cool off, but the damage output may be worth it.",
```
Comment (TheShadowFerret) [inline data/mods/Xedra_Evolved/items/inventor/gunmods.json:299]:
```suggestion
    "description": "Replace a large portion of the radiator with additional capacitors.  The gun will barely cool off, but the damage output may be worth it.",
```

File: data/mods/Xedra_Evolved/items/inventor/gunmods.json [hunk 5]
```diff
@@ -165,5 +165,149 @@
...
+    "location": "heat sink",
+    "mod_targets": [ "pistol", "rifle", "smg" ],
+    "cooling_value_multiplier": 0.75,
+    "min_skills": [ [ "deduction", 4 ], [ "weapon", 4 ] ]
+  },
+  {
+    "id": "inventor_lasmod_adv_adv_cooler",
+    "type": "GUNMOD",
+    "name": { "str_sp": "freezing heat sink" },
+    "description": "Small piece of completely frozen metal.  Grant a massive cooling improvement, but the weapon circuits don't work properly with this mod, making you spend more energy per shot.",
```
Comment (TheShadowFerret) [inline data/mods/Xedra_Evolved/items/inventor/gunmods.json:190]:
```suggestion
    "description": "Small piece of completely frozen metal.  It grants a massive cooling improvement, but the weapon circuits don't work properly with this mod, making you spend more energy per shot.",
```

**Comment by CHCOOH** [comment]
I think you are using cooling_value_multiplier incorrectly.
If you want to improve cooling performance, you need to set a value higher than 1, such as 1.2.

Come to think of it, the gun
"overheat_threshold"
"cooling_value"
"heat_per_shot"
I did not add an explanation regarding.
Should I add them to the GUNMOD in JSON_INFO.md, such as "the more it increases, the better it performs"?

**Comment by GuardianDll** [comment]
> I think you are using cooling_value_multiplier incorrectly.

oh, i did, thanks for pointing it out

> I did not add an explanation regarding.

No, you don't need to document it, i documented it before, it's in gun definition:

https://github.com/CleverRaven/Cataclysm-DDA/blob/32e562d880a3396c45ca2530752b226e9de45402/doc/JSON_INFO.md?plain=1#L3899-L3901

**Pull request merged**